packages/orchestration/test/staking-ops.test.ts

```diff
@@ -228,6 +228,7 @@ const makeScenario = () => {
 
   const baggage = makeScalarBigMapStore('b1') as Baggage;
   const zone = makeDurableZone(baggage);
+  // TODO: wrap marshaller for fidelity?
```

Comment:
```suggestion
  // XXX: could wrap marshaller for higher fidelity
```

or is that TODO meant to be done in this PR?
Or are we opening an issue to do it?